stdlib/public/SDK/Foundation/AffineTransform.swift

```diff
@@ -280,16 +292,22 @@ public struct AffineTransform : ReferenceConvertible, Hashable, CustomStringConv
     public var debugDescription: String {
         return description
     }
-
+    
     public static func ==(lhs: AffineTransform, rhs: AffineTransform) -> Bool {
         return lhs.m11 == rhs.m11 && lhs.m12 == rhs.m12 &&
-               lhs.m21 == rhs.m21 && lhs.m22 == rhs.m22 &&
-               lhs.tX == rhs.tX && lhs.tY == rhs.tY
```

Comment:
The indentation changes are from copy/paste from Xcode. Imho all overlay code should be 100% copy paste-able from the swift-corelibs-foundation Xcode project to the Foundation overlay.